packages/react-devtools-shell/src/app/console.js

```diff
@@ -26,7 +20,12 @@ function ignoreStrings(
         }
       }
     }
-    originalMethod(...args);
+
+    // HACKY In the test harness, DevTools overrides the parent window's console.
+    // Our test app code uses the iframe's console though.
+    // To simulate a more accurate end-to-end environment,
+    // the shell's console patching should pass through to the parent override methods.
+    window.parent.console[methodName](...args);
```

Comment:
Ooo, what did moving this fix? 😃

Reply:
The `ignoreStrings` function would run once and store the OG console method in `originalMethod`. However we patch and unpatch the console for strict mode multiple times, but this change was getting ignored becuase we were never calling `window.parent.console`. We were calling `originalMethod` which was `window.parent.console` at some previous point in time.